aws/resource_aws_api_gateway_integration.go

```diff
@@ -240,6 +269,8 @@ func resourceAwsApiGatewayIntegrationRead(d *schema.ResourceData, meta interface
 	d.Set("request_parameters", aws.StringValueMap(integration.RequestParameters))
 	d.Set("request_parameters_in_json", aws.StringValueMap(integration.RequestParameters))
 	d.Set("passthrough_behavior", integration.PassthroughBehavior)
+	d.Set("connection_type", integration.ConnectionType)
```

Comment:
Looks like this might not always be returned? Maybe we should set it to `INTERNET` if nil

```
--- FAIL: TestAccAWSAPIGatewayIntegrationResponse_basic (110.48s)
    testing.go:518: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: aws_api_gateway_integration.test
          connection_type: "" => "INTERNET"
```

Reply:
Updated but a bit curious though.
I think I covered the attribute check in `TestAccAWSAPIGatewayIntegration_integrationType`. Is "not always" behavior caused by regional difference from aws api? If so, it looks tricky to discover.